src/common/components/nav/nav.bundle.js

```diff
@@ -7,11 +7,11 @@
   // angular.js to load itself into
   const angular = (window.angular = {});
 
-  require('./nav.component');
+  const nav = require('./nav.component');
```

Comment:
Ya it works. I tested it after your comment. When requiring the component it runs all the nav code and all the calls to `angular.xxx` add modules/components/whatever to the global angular window object which gets bootstrapped with the code here. This is the same as the `import`s in the rest of the app. This is no different than before other than I am now also using the default export to get the module name.

I guess I really didn't need to touch this file but at first I thought I might need to import the analytics module here. I guess this just makes it more consistent with the other angular module imports.